internal/cli/cmd/install.go

```diff
@@ -69,7 +69,8 @@ cilium install --context kind-cluster1 --cluster-id 1 --cluster-name cluster1
 	cmd.Flags().DurationVar(&params.WaitDuration, "wait-duration", 15*time.Minute, "Maximum time to wait for status")
 	cmd.Flags().BoolVar(&params.RestartUnmanagedPods, "restart-unmanaged-pods", true, "Restart pods which are not being managed by Cilium")
 	cmd.Flags().BoolVar(&params.Encryption, "encryption", false, "Enable encryption of all workloads traffic")
-	cmd.Flags().BoolVar(&params.NodeEncryption, "node-encryption", false, "Enable encryption of all node to node traffic")
+	cmd.Flags().StringVar(&params.EncryptionMode, "encryption-mode", "ipsec", "Encryption mode { ipsec | wireguard }")
```

Comment:
I'm also fine to change the `--encryption` flag to be a string. The fewer flags the better. We are still pre 1.0.